helm-frontend/src/components/SimpleHero.tsx

```diff
@@ -0,0 +1,36 @@
+import helmHero from "@/assets/helmhero.png";
+import { Link } from "react-router-dom";
+
+export default function SimpleHero() {
+  return (
+    <div className="flex px-6 py-20">
+      <div className="flex-1 p-4 flex flex-col justify-center">
+        <div className="flex justify-start">
+          <h1 className="text-5xl mb-4 mx-4 mt-2">
+            <strong>
+              A holistic framework for evaluating foundation models.
+            </strong>
+          </h1>
+        </div>
+        <div className="flex justify-start mt-6 ml-4">
+          <Link to="leaderboard">
```

Comment:
Not sure that this makes sense (unless we consider "Lite" as the "default" HELM project) but let's merge this first and discuss later.

Reply:
Agree, my plan is to have a follow-up PR to add a trimmed header for the Global page and replace these buttons with "Our Projects" and "Github"